tests/post/features/volume.feature

```diff
@@ -11,15 +11,6 @@ Feature: Volume management
 
     Scenario: Test volume creation (sparseLoopDevice)
         Given the Kubernetes API is available
-        When I create the following Volume:
-            apiVersion: storage.metalk8s.scality.com/v1alpha1
-            kind: Volume
-            metadata:
-              name: volume1
-            spec:
-              nodeName: bootstrap
-              storageClassName: metalk8s-prometheus
-              sparseLoopDevice:
-                size: 10Gi
+        When I create a Volume 'volume1'
```

Comment:
No, when you create a volume you want to specify the body.

This should be left untouched

Reply:
This is not resolved: you didn't restore the step as it was…